backend/src/controllers/sessionsControllers.ts

```diff
@@ -0,0 +1,49 @@
+
+import { Request, Response } from 'express';
+import { createSessionService } from '../services/sessionsServices';
+import jwt from 'jsonwebtoken';
+
+
+export const createSessionController = async(req: Request, res: Response) => {
+
+    try {
+
+        const result = await createSessionService()
+
+        const currentUserAccessToken = result.currentUserAccessToken.access_code
+        const guestUserAccessToken = result.guestUserAccessToken.access_code
+
+        const jwtExpiry = 86400;
+
+        const jwtToken = await jwt.sign({ key: currentUserAccessToken }, process.env.JWT_SECRET, {
+          expiresIn: jwtExpiry
+        });
+    
+        res.cookie('jwt', jwtToken, {
+          httpOnly: true,
+          sameSite: 'strict',
+          maxAge: jwtExpiry * 1000 // maxAge is in MS
+        });
+
+        return res.status(201).json({
+            success: true,
+            message: 'ok',
+            data: {
+                tokens: {
+                    currentUserAccessToken,
+                    guestUserAccessToken   
+                },
+            }
+            });
+        
+    } catch (error) {
+
+        return res.status(201).json({
```

Comment:
the status here should be 500. If you are not going to add details of the errore in the `message` field in the future, I woul remove it, as it does not add more info to the client.

Reply:
Fixed.